client/src/App.js

```diff
@@ -1,16 +1,24 @@
 import { Route, Switch } from "react-router-dom";
 import Login from "./pages/Login";
 import Home from "./pages/Home";
+import Calendar from "./pages/Calendar";
+import NavbarComponent from "./Component/NavbarComponent";
 
 const App = () => (
-	<Switch>
-		<Route path="/" exact>
-			<Home />
-		</Route>
-		<Route path="/login">
-			<Login />
-		</Route>
-	</Switch>
+	<Route>
+		<NavbarComponent />
+		<Switch>
+			<Route path="/" exact>
+				<Home />
+			</Route>
+			<Route path="/login">
+				<Login />
+			</Route>
+			<Route path="/calendar">
+				<Calendar />
+			</Route>
+		</Switch>
```

Comment:
```suggestion
		<Container>
			<Switch>
				<Route path="/" exact>
					<Home />
				</Route>
				<Route path="/login">
					<Login />
				</Route>
				<Route path="/calendar">
					<Calendar />
				</Route>
			</Switch>
		</Container>
```
Since we're here, let's wrap all pages in a `Container` so we add left and right margin to all pages. We can then delete the `container` class in the `Home` and `Calendar` components

Reply:
Ok that's fantastic.